native/elixir/patches/001-makefile.patch

```diff
@@ -0,0 +1,17 @@
+--- Makefile.orig	2019-05-11 14:29:01.000000000 +0200
++++ Makefile	2021-02-10 13:07:34.283425013 +0100
+@@ -1,4 +1,4 @@
+-PREFIX ?= /usr/local
++PREFIX ?= /usr/local/elixir
```

Comment:
Same as above.

with the following definitions in the Makefile there is no patch needed (needs `DEPENDS = native/erlang`)

``` make

ERLANG_BIN_DIR = $(realpath $(WORK_DIR)/../../../native/erlang/work-native/install/usr/local/bin)

CONFIGURE_TARGET = nop
COMPILE_TARGET = elixir_compile
INSTALL_TARGET = elixir_install

include ../../mk/spksrc.native-cc.mk

.PHONY: elixir_compile
elixir_compile:
	$(RUN) PATH=$$PATH:$(ERLANG_BIN_DIR) $(MAKE) $(MAKE_COMPILE_OPTIONS)

.PHONY: elixir_install
elixir_install:
	$(RUN) PATH=$$PATH:$(ERLANG_BIN_DIR) $(MAKE) DESTDIR=$(INSTALL_DIR) PREFIX=$(INSTALL_PREFIX) install
```